### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by dr4nkle4n** [seerge/g-helper#2474] [General]
Do I need G-Helper running in the background for it to set the clock speed boosts and what not? I know it is super light weight so there is no point in even not running it, but it won't run on start up for some reason and I can't feel bothered to open it every time. With that being said, do I need G-Helper running?

**Comment by Kikimahe** [comment]
Hi,

The bios is reset at startup. So you need Ghelper at startup.

Did you check “Run at Startup” in Ghelper ?
Check in Task Scheduler that the Ghelper task has the correct path to **your** location of the ghelper.exe file.

![Sans titre-1](https://github.com/seerge/g-helper/assets/142881210/d727482a-557a-4646-825d-62f7508ebf31)



Another startup solution would be to place a Ghelper.exe shortcut in this folder :
`C:\Users\yourname\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup\`

**Reply by dr4nkle4n** [reply]
So all the clocks reset so I need it running? I figured it out to run at start up but I am still confused if I need it open in the background

**Reply by Kikimahe** [reply]
If you are not using Ghelper, you must change manually  the 3 Windows power modes.
`\ Settings \ Battery and Power \ Power Mode`

- Best Power Efficiency (= silent in Ghelper)
- Balanced (= balanced)
- Best performance (= turbo)

**Comment by seerge** [comment]
@dr4nkle4n hello,

You don't need G-Helper for clocks, as it doesn't set any clocks in a first place and doesn't manage your hardware in realtime anyhow.

However you **DO NEED** G-Helper running in order to process hotkeys or do automatic mode switching on battery / plugged in, etc. So I advice to figure out why exactly you can't run app on satartup.

Try to run it as admin and uncheck / check "run on startup" checkbox, it would re-create a task again for you

**Reply by dr4nkle4n** [reply]
Can't you set a mem boost and force the clock of the cpu? If I did these do I still need G-Helper running in the background?

**Reply by seerge** [reply]
I don't know what do you mean under "mem boost" and "force the clock", but neither G-Helper nor the AC do that. 

``CPU Boost`` is a setting in windows power plan, and it remains same no matter if you are running app or not.

**Reply by dr4nkle4n** [reply]
g-helper has an option to change cpu clock speeds and gpu memory

**Reply by seerge** [reply]
@dr4nkle4n GPU clock offset and memory offset remain until next reboot.
